client/components/TestReview/index.jsx

```diff
@@ -0,0 +1,270 @@
+                                    )}.`}
+                            </li>
+                        )}
+                    </Ul>
+                </li>
+                <li>
+                    <strong>Commit:&nbsp;</strong>
+                    {testPlanVersion.gitMessage}
+                </li>
+            </ul>
+            <h2>List of Tests</h2>
+            <FilterButtonContainer>
+                <FilterButtons
+                    filterLabel="Filter tests by AT"
+                    filterOptions={filterOptions}
+                    activeFilter={activeFilter}
+                    onFilterChange={selectedFilter => {
+                        setActiveFilter(selectedFilter);
+                    }}
+                />
+            </FilterButtonContainer>
+            {filteredTests.map((test, index) => {
+                const isFirst = index === 0;
+
+                const atMode =
+                    test.atMode.substring(0, 1) +
+                    test.atMode.toLowerCase().substring(1);
+
+                const specifications =
+                    test.renderableContents[0].renderableContent.info.references.map(
+                        ({ refId, value }) => [refId, value]
+                    );
+
+                let filteredAts;
+                if (activeFilter === 'All ATs') {
+                    filteredAts = test.ats;
+                } else {
+                    filteredAts = test.ats.filter(
+                        at => at.name === activeFilter
+                    );
+                }
+
+                return (
+                    <Fragment key={test.id}>
+                        {isFirst ? null : <hr />}
+                        <h3>{`Test ${test.rowNumber}: ${test.title}`}</h3>
+                        <ul>
+                            <li>
+                                <strong>Mode:&nbsp;</strong>
+                                {atMode}
+                            </li>
+                            <li>
+                                <strong>Assistive technologies:&nbsp;</strong>
+                                {test.ats.map(at => at.name).join(', ')}
+                            </li>
+                            <li>
+                                <strong>Relevant specifications:&nbsp;</strong>
+                                {specifications.map(([title, link], index) => {
+                                    const isLast =
+                                        index === specifications.length - 1;
+                                    return (
+                                        <Fragment key={title}>
+                                            <a
+                                                href={link}
+                                                rel="noreferrer"
+                                                target="_blank"
+                                            >
+                                                {title}
```

Comment:
This produces link text such as `example`, `aria-live`, etc which is different from what we have now being `APG example: <pattern>.html`, `ARIA specification: aria-live`, etc.

I think this is fine for now given how the app imports those tests, but just calling out that this will be extended in the future to [support clearly defined link text for v2 test format tests](https://github.com/w3c/aria-at/wiki/Test-Format-Definition-V2#referencescsv).

There is no request for a change here, just a note.